doc/lua_api.txt

```diff
@@ -6302,9 +6304,13 @@ Misc.
     * `method` is a string identifying the compression method to be used.
     * Supported compression methods:
         * Deflate (zlib): `"deflate"`
+        * Zstandard: `"zstd"`
     * `...` indicates method-specific arguments. Currently defined arguments
       are:
         * Deflate: `level` - Compression level, `0`-`9` or `nil`.
+        * Zstandard: `level` - Compression level. Integer or `nil`.
+        Note any supported Zstandard compression level could be used here,
+        but these are subject to change between Zstandard versions.
 * `minetest.decompress(compressed_data, method, ...)`: returns data
     * Decompress a string of data (using ZLib).
```

Comment:
```suggestion
    * Decompress a string of data using the algorithm specified by `method`.
```

Reply:
Done as of [b5bdea9](https://github.com/minetest/minetest/pull/12515/commits/b5bdea98418d1c2abc44904e83c8b7ec4ba386b3).